assets/js/admin/job-tags-upsell.js

```diff
@@ -0,0 +1,25 @@
+import { registerPlugin } from '@wordpress/plugins';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { ExternalLink } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const PROMO_LINK = 'https://wpjobmanager.com/add-ons/job-tags/?utm_source=plugin_wpjm&utm_medium=upsell&utm_campaign=job_tags_editor_upsell';
+
+const WPJM_Job_Tags_Upsell_Panel = () => (
+    <PluginDocumentSettingPanel
+        name="job-tags-upsell-panel"
+        title="Job Tags"
```

Comment:
I think we might want to make this translatable? I think on the WPJM Tags plugin it is.

```suggestion
        title={ __( 'Job Tags', 'wp-job-manager' ) }
```